jetty-io/src/main/java/org/eclipse/jetty/io/ManagedSelector.java

```diff
@@ -160,22 +145,132 @@ protected void doStop() throws Exception
         super.doStop();
     }
 
+    void compute(Selector selector, SelectableChannel channel, SelectionKey selectionKey, Consumer<SelectionKey> action)
```

Comment:
`compute` is not really very meaningful

Reply:
What then? It runs the `Consumer` given those parameters - similar to `Map.compute()`.